docs/api/python/z3compat/array.rst

```diff
@@ -1,2 +1,43 @@
 Arrays
 ============
+
+
+Basic Array Term Builders
+-------------------------
+
+.. autofunction:: cvc5_z3py_compat.Array
+.. autofunction:: cvc5_z3py_compat.K
```

Comment:
Why is this called `K`? Do we have an alias for this with a reasonable name?

Reply:
I've added an alias.